python/hsfs/feature_group.py

```diff
@@ -3290,6 +3290,8 @@ def insert(
         group.
         If feature group's time travel format is `HUDI` then `operation` argument can be
         either `insert` or `upsert`.
+        For feature groups with time travel format `DELTA`, `delta.enableChangeDataFeed` option can be set in write_options. If
```

Comment:
putting the option in insert gives user an impression that it is configurable in each insert, while I think it's not how it works.